skadi-lib/build.gradle.kts

```
@@ -21,3 +20,73 @@ dependencies {
     testImplementation("com.google.truth:truth:${Dependencies.truth}")
     testImplementation("com.github.ologe:flow-test-observer:1.4.0")
 }
+
+val siteUrl = "https://github.com/Syex/skadi"
+val vcsUrl = "https://github.com/Syex/skadi.git"
+val libDescription = "A Kotlin JVM library featuring a redux-like architecture with coroutines"
+
+val libVersion = "0.1.0"
+
+// maven publish plugin
+val sourcesJar by tasks.registering(Jar::class) {
+    archiveClassifier.set("sources")
+    from(sourceSets.main.get().allSource)
+}
+
+publishing {
+    publications {
+        create<MavenPublication>("skadi") {
+            group = "de.syex"
+            artifactId = "skadi"
+            version = libVersion
+            from(components["java"])
+            artifact(sourcesJar.get())
+
+            pom {
+                name.set("skadi")
+                description.set(libDescription)
+
+                licenses {
+                    license {
+                        name.set("The Apache License, Version 2.0")
+                        url.set("http://www.apache.org/licenses/LICENSE-2.0.txt")
+                    }
+                }
+
+                developers {
+                    developer {
+                        id.set("Syex")
+                        name.set("Tom Seifert")
+                    }
+                }
+
+                scm {
+                    connection.set(vcsUrl)
+                    developerConnection.set(vcsUrl)
+                    url.set(siteUrl)
+                }
+            }
+        }
+    }
+}
+
+// bintray plugin
+bintray {
+    user = System.getenv("bintrayUser")
+    key = System.getenv("bintrayApiKey")
+    setPublications("skadi")
+    override = true
+    pkg = PackageConfig().apply {
```

I guess it is saver to use `pkg.apply{}` here instead of creating a new config..
```suggestion
    pkg.apply {
```